docs/internal-documentation/maintenance/SupportCriteria.md

```diff
@@ -0,0 +1,124 @@
+Support criteria
+================
+This document hosts a set of guiding principles for selecting configuration
+parameter items to support.
+
+
+## General
+
+It is legitimate to avoid supporting items that are otherwise perfectly eligible
+for support, just for the sake of limiting the total number of configuration
+tuples.
+
+
+## Architectures
+
+Can we say something about how we chose to support only amd64/x86_64 and nothing
+else?
+
+
+## Operating systems
+
+Can we say something about how the current quartet of operating systems was
+chosen?
+
+
+## Operating system versions
+
+Minor version/point releases should be specified. Patch levels should not be specified.
+
+Operating system versions without long term support form their vendor should not be supported.
+
+Operating system versions that have reached their end-of-life should not be supported.
+
+Operating system versions that do not provide the prerequisites for Zonemaster should not be supported.
+
+Operating system specific guidelines:
+
+* CentOS:
+  * Base Distributions are listed here:
+    https://en.wikipedia.org/wiki/CentOS#End-of-support_schedule
+  * The current minor releases are listed at:
+    https://wiki.centos.org/Download
+
+* Debian:
+  * Current versions of "stable" and "oldstable" are listed here:
+    https://wiki.debian.org/DebianReleases#Current_Releases.2FRepositories
+  * The current point releases are listed here:
+    https://wiki.debian.org/DebianReleases/PointReleases
+
+* FreeBSD:
+  * Extended releases are listed here:
+    https://www.freebsd.org/security/
+  * Patch level should not be specified.
+
+* Ubuntu:
+  * LTS releases are listed here:
+    https://wiki.ubuntu.com/Releases
+  * The supported patch level is the one specified at:
+    https://wiki.ubuntu.com/Releases
+
+
+## Database engines
+
+Can we say something about how the current triple of database engines was
+chosen?
```

Comment:
I think that we should only state that Zonemaster backend only is integrated with the selected databases. I do not think that we have to motivate the choice.